contrib/charts/pebble/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```

Comment:
Update the description, unless this has been imported from another project.

Reply:
This chart isn't used by end users, only for e2e tests. It's the bare minimum to deploy pebble, so I'm not bothering to fix up this chart.